Specs/Sequence/Chinese/URLModel.json

```diff
@@ -0,0 +1,251 @@
+        "Resolution": {
+          "value": "http://abc.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "www.abc.com",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "www.abc.com",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "www.abc.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "www.abc.com.cn",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "www.abc.com.cn",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "www.abc.com.cn"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "abc.com",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "abc.com",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "abc.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com/file_name",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com/file_name",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com/file_name"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com/file_path/",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com/file_path/",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com/file_path/"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com/file_path/123.html",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com/file_path/123.html",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com/file_path/123.html"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com/search?id=123",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com/search?id=123",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com/search?id=123"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com/search?id=123&name=Alice",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com/search?id=123&name=Alice",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com/search?id=123&name=Alice"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com/123#cite_1",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com/123#cite_1",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com/123#cite_1"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "http://abc.com:8080",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "http://abc.com:8080",
+        "TypeName": "url",
+        "Resolution": {
+          "value": "http://abc.com:8080"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "网址是microsoft.com",
```

Comment:
Many test cases are missing here when compared to the English spec. Can you add the others and translate them when needed?
Of course you can leave all the new ones here too. Thanks!